taxonomies/food/ingredients.txt

```diff
@@ -2631,13 +2607,13 @@ it: burro di capra
 # description:en:MILK - a white liquid nutrient-rich food produced by the mammary glands of mammals.
 
 < en:dairy
-en: milk
+en: milk, milk ingredients
```

Comment:
What kind of products have "milk ingredients" as an ingredient? that seems  very strange